README.md

```diff
@@ -234,17 +234,6 @@ tested version during CI validation.
 
 ## Nightly builds
 
-The Scala CI builds nightly download releases (including all modules) and publishes
-them to the following locations:
-  - [2.12.x](http://www.scala-lang.org/files/archive/nightly/2.12.x/?C=M;O=D)
-  - [2.13.x](http://www.scala-lang.org/files/archive/nightly/2.13.x/?C=M;O=D)
-
-The CI also publishes nightly API docs:
-  - [2.12.x](http://www.scala-lang.org/files/archive/nightly/2.12.x/api/?C=M;O=D)
-    - [symlink to the latest](http://www.scala-lang.org/files/archive/nightly/2.12.x/api/2.12.x/)
-  - [2.13.x](http://www.scala-lang.org/files/archive/nightly/2.13.x/api/?C=M;O=D)
```

Comment:
Scaladocs here https://www.scala-lang.org/files/archive/nightly/2.13.x/api/2.13.x/ link to 2.13.0-pre-81ad83d which was released February 1st 2018.

Reply:
yeah, https://github.com/scala/scala-lang/issues/929 :-/